### DISCUSSION TITLE
Inline edit directly in table cell

### DISCUSSION THREAD

**Discussion by studiowizjo** [filamentphp/filament#3044] [Ideas]
For some time I think that there would be nice if there will be possibility to quick, inline edit some fields strictly from table level, without need to go inside each record. This could work nice for boolean values which can be toggled, for some statuses, which you could choose from the select, short texts and so on.

There could be 2 approaches for this to happen:
1. click (maybe double click?) value and it will replace with form field
2. click value and it will open modal to edit

Personally, I think the 1st solution would be nicer, but maybe there are some downsides I don`t see for now. One of them is that it can stretch row, but I'm not entirely sure if this could be a problem.

What do you think?

**Comment by acharseth** [comment]
Agree, capability for inline editing would be really usefull. And I think option 1 is the most elegant and user friendly option.

**Reply by cirrion-software** [reply]
You can [here](https://filamentphp.com/docs/3.x/tables/columns/getting-started) have a look at the column types

**Reply by micraux** [reply]
Try a [text-input column](https://filamentphp.com/docs/3.x/tables/columns/text-input)

**Reply by juanborras** [reply]
Anyway to disable autosave on TextInputColumn?
